ci-operator/step-registry/agent-qe/baremetal/install/ove/disconnected/mirror-operators/agent-qe-baremetal-install-ove-disconnected-mirror-operators-commands.sh

```
@@ -0,0 +1,479 @@
+function disable_default_catalogsource () {
+    run_command "oc patch operatorhub cluster -p '{\"spec\": {\"disableAllDefaultSources\": true}}' --type=merge"; ret=$?
+    if [[ $ret -eq 0 ]]; then
+        echo "disable default Catalog Source successfully."
+    else
+        echo "!!! fail to disable default Catalog Source"
+        return 1
+    fi
+    ocp_version=$(oc get -o jsonpath='{.status.desired.version}' clusterversion version)
+    major_version=$(echo ${ocp_version} | cut -d '.' -f1)
+    minor_version=$(echo ${ocp_version} | cut -d '.' -f2)
+    if [[ "X${major_version}" == "X4" && -n "${minor_version}" && "${minor_version}" -gt 17 ]]; then
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)